### ISSUE TITLE
Error on requiring faker library 

### ISSUE THREAD

**Issue by rollymaduk** [Marak/faker.js#430]
Hi,
Please any ideas on how to fix this error below
`node_modules\faker\vendor\password-generator.js:19Uncaught TypeError: Cannot read property 'localPasswordGeneratorLibraryName' of undefined(…)`
in my browser console when I try to load my page with faker.js library, I am using node and browserfy.

**Comment by mbeaudru** [comment]
I am encountering the same problem and I can't figure out why I have this error.

Installed with ```yarn add faker``` and importing, this same error pops out.

Did you figured this out ?

**Comment by rollymaduk** [comment]
Hi @mbeaudru , I fixed this by adding a null checker in the affected file on line 19.
``` js  
/// changed the line below from
localName = root.localPasswordGeneratorLibraryName || "generatePassword",
//to 
 localName = root && root.localPasswordGeneratorLibraryName || "generatePassword",
```
I have made a PR #445  that addresses this.

**Comment by rollymaduk** [comment]
@mbeaudru  please let me know if it solved the problem so I can close this issue

**Comment by canhkieu** [comment]
Same issues, I was changed to but still error/
 localName = root && root.localPasswordGeneratorLibraryName || "generatePassword",

**Comment by rollymaduk** [comment]
@canhkieu  how did you change it? did you fork the project and use npm link to locally test it? if you did that, then ensure you also removed your global and project modules by either using npm uninstall <package> -g or npm uninstall <package> -D or npm uninstall <package> --save,
you can also post your error logs to help me get a better picture of what is happening!
thanks

**Comment by Marak** [comment]
@rollymaduk -

Does this actually solve the issue? What is the cause of the issue?

Not sure why this is happening. I'll see if I can investigate further.

**Comment by Marak** [comment]
I've opted to move the password generation logic to faker core. The extra browserifying type logic in `vendor/password-generator.js` was causing issues.

Should be resolved in `master`.

Also resolves #460
